data/persons.txt

```diff
@@ -2035,3 +2035,175 @@ outfit "Emotional Baggage"
 	thumbnail "outfit/unknown"
 	description `The physical embodiment of everything that makes the Reviewers who they are today.`
 	description `	Unfortunately, it's not for sale.`
+
+
+
+person "Danau"
+	government "Author"
+	frequency 100
+	personality
+		heroic unconstrained opportunistic
+	phrase
+		word
+			"When you want to stop your ship, press shift and reverse."
+			"Your hyperdrive doesn't work? Have you tried turning it off and on again?"
+	phrase
+		word
+			"Change the orientation of your"
+		word
+			" "
+		word
+			"drone"
+			"interceptor"
+			"fighter"
+		word
+			" "
+		word
+			"bays"
+			"docking ports"
+		word
+			" "
+		word
+			"just for the fun of it."
+			"for faster deployments."
+			"on your new Marauder ship."
+			"to improve atmospheric handling."
+	phrase
+		word
+			"I have a delivery for"
+			"I'm looking for"
+		word
+			" "
+		word
+			"Arrakis"
+			"Caladan"
+			"the Eaxaw"
+			"the Gyisache"
+			"the Happy Birthday"
+			"the Laenfa"
+			"the Muursh"
+			"the Ourbools"
+			"Tycho Magnetic Anomaly 2"
+		word
+			"; "
+		word
+			"is this the right game?"
+	phrase
+		word
+			"The Hauler VI is a nice ship, "
+		word
+			"as long as you don't activate the cargo refrigeration systems."
+			"but it hasn't gotten beyond the prototype stage yet."
+			"but it is not for sale on the regular market."
+			"but using two Hauler IIIs is probably more economical."
+	phrase
+		word
+			"Automate"
+			"Extend"
+			"Improve"
+			"Streamline"
+		word
+			" "
+		word
+			"our"
+			"the ES"
+			"your"
+		word
+			" "
+		word
+			"builds"
+			"builds and tests"
+			"CI jobs"
+			"coding style checks"
+			"integration processes"
+			"merge process"
+			"software tests"
+			"test and integration processes"
+			"test processes"
+			"tests"
+		word
+			"."
+			"!"
+		word
+			" "
+		word
+			"Avoid crashes of the game!"
+			"Improve the quality of our codebase!"
+			"Get new features into the game faster!"
+			"Have more time for good reviews!"
+			"Have more time for playing the game!"
+			"Prevent bugs!"
+
+	ship "Modified Hauler VI" "PTR - HVI 42"
+	ship "Hauler III" "PTR - Cliff"
```

Comment:
Person esort-ships are typically restored after they get destroyed (if the flagship survives). What happens if one of those escorts gets captured, while the flagship isn't destroyed? Would the restore just move a captured ship back to the author government?

Reply:
Just tested this; captured ships are indeed set back to the author government in this situation (giving the situation where your escorts suddenly become author-ships, and obviously attacking you, since the capture wasn't appreciated.)

Reply:
Saving and reloading the game fixes this by creating a copy, but the issue still exists when you don't do a reload between capturing and encountering the author ship again.

Reply:
I created an issue for this: https://github.com/endless-sky/endless-sky/issues/10379. We might want to make all the escort ships `"never captured"` for now though.

Reply:
I applied the `"never disabled"` workaround for now on those ships. (Which means that they last a little bit longer in battle, but it shouldn't make much of a difference.)